integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -154,31 +159,31 @@ struct TestReshardingParameters {
 
 impl TestReshardingParameters {
     fn new() -> Self {
-        Self::with_clients(3)
+        Self::with_validators(3)
     }
 
-    fn with_clients(num_clients: u64) -> Self {
+    fn with_validators(num_validators: u64) -> Self {
         let num_accounts = 8;
         let initial_balance = 1_000_000 * ONE_NEAR;
         let epoch_length = 6;
         let track_all_shards = true;
         let all_chunks_expected = true;
 
         // #12195 prevents number of BPs bigger than `epoch_length`.
-        assert!(num_clients > 0 && num_clients <= epoch_length);
+        assert!(num_validators > 0 && num_validators <= epoch_length);
 
         let accounts = (0..num_accounts)
             .map(|i| format!("account{}", i).parse().unwrap())
             .collect::<Vec<AccountId>>();
 
-        // This piece of code creates `num_clients` from `accounts`. First client is at index 0 and
-        // other clients are spaced in the accounts' space as evenly as possible.
-        let clients_per_account = num_clients as f64 / accounts.len() as f64;
-        let mut client_parts = 1.0 - clients_per_account;
-        let clients: Vec<_> = accounts
+        // This piece of code creates `num_validators` from `accounts`. First validator is at index 0 and
+        // other validator are spaced in the accounts' space as evenly as possible.
+        let validators_per_account = num_validators as f64 / num_accounts as f64;
+        let mut client_parts = 1.0 - validators_per_account;
+        let block_and_chunk_producers: Vec<_> = accounts
```

Comment:
What about chunk validators?

Reply:
Just refactored it a little bit in order to add archival node. Will add TODO to test chunk validators as well.